### DISCUSSION TITLE
What is the replacement for PostgreSQLDataType?

### DISCUSSION THREAD

**Discussion by Gustl22** [isoos/postgresql-dart#281] [General]
In v3 PostgreSQLDataType was removed. In the deprecation message is no replacement stated.

[Here](https://github.com/isoos/postgresql-dart/pull/165) the PgDataType was introduced, but the file was also removed.

What is the according alternative? And one may can include this in the deprecation message.

**Comment by Gustl22** [comment]
Ok I found it, it's simply `Type`.

**Comment by isoos** [comment]
https://pub.dev/documentation/postgres/latest/postgres/Type-class.html mostly, but it is not a 100% replacement, IIRC the old one had a few methods that are not recommended to be used.
